### ISSUE TITLE
Question:  Why the manual Perl compilation?

### ISSUE THREAD

**Issue by billdenney** [dpastoor/dockerfiles#1]
When comparing your PsN installation to the one that I use, you appear to manually compile perl from sources.  What is the benefit?  (And generally, should I be doing the same?  I've had no issues to date.)

**Comment by dpastoor** [comment]
I can't remember the exact error I ran into - but basically I could not get the some of the libs to install correctly. I couldn't use yours (as it points to ubuntu registry) and I had decided to go with debian vs ubuntu for "stability" and a little smaller image size.

Ultimately I just said screw it just went and grabbed a perl container for debian, and took the setup components. When using that, everything installed correctly. I wasn't too concerned with the extra build time, as I'm not re-building often.

This container is super minimal effort as I just wanted it available.  I actually think psn is overkill for my use cases and am focusing way more effort into https://github.com/dpastoor/nonmemutils to replace the standard execute/scm/sse tasks. Since its written in go, its just ~10mb binary. 

So to the point - I would not use this as a canonical reference at all.
